### PR TITLE
Add missing tooltips for KMS advanced modal

### PR DESCRIPTION
**Pull request by a2batic**
Signed-off-by: Kanika Murarka <kmurarka@redhat.com>

### CODE REVIEW

**Comment by afreen23** [comment]
Please run yarn i18n to update the keys.

File: frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.tsx [hunk 1]
```diff
@@ -235,9 +235,7 @@ const KMSDetails: React.FC<KMSDetailsProps> = ({ setEditKMS, currentKMS }) => {
           </TextListItem>
           <TextListItem>
             {t('ceph-storage-plugin~Provider:')}{' '}
-            <span className="help-block ocs-storageClass-encryption__help-block">
-              {currentKMS?.KMS_PROVIDER}
-            </span>
+            <span className="help-block ocs-storageClass-encryption__help-block">Vault</span>
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.tsx:238]: externalize this string - 'Vault'

**Review by afreen23 — CHANGES_REQUESTED** [review]

File: frontend/packages/ceph-storage-plugin/src/components/modals/advanced-kms-modal/advanced-kms-modal.tsx [hunk 2]
```diff
@@ -53,13 +53,29 @@ export const AdvancedKMSModal = withHandlePromise((props: AdvancedKMSModalProps)
   const [error, setError] = React.useState('');
 
   const vaultNamespaceTooltip = t(
-    'ceph-storage-plugin~Vault enterprise namespaces are isolated environments that functionally exist as Vaults within a Vault. They have separate login paths and support creating and managing data isolated to their namespace.',
+    'ceph-storage-plugin~The Vault namespace to use for the OpenShift Container Storage communication with the Vault server. Using relative paths is allowed. Mainly used for KMS tenants.',
```
Comment (nixpanic) [inline frontend/packages/ceph-storage-plugin/src/components/modals/advanced-kms-modal/advanced-kms-modal.tsx:56]: This reads a little difficult. The previous contents made more sense to me.

Note that this is not about OpenShift Namespaces, but Namespaces inside the Vault service (only available with Hashicorps Vault Enterprise product). More details are at https://www.vaultproject.io/docs/enterprise/namespaces

I do not know about the request to change this text, so you do not need to see this as a _needs to change_ review.

File: frontend/packages/ceph-storage-plugin/src/components/modals/advanced-kms-modal/advanced-kms-modal.tsx [hunk 3]
```diff
@@ -53,13 +53,29 @@ export const AdvancedKMSModal = withHandlePromise((props: AdvancedKMSModalProps)
...
   );
 
   const KMSFileSizeErrorMsg = t(
     'ceph-storage-plugin~Maximum file size exceeded. File limit is 4MB.',
   );
 
+  const vaultCACertTooltip = t(
+    `ceph-storage-plugin~A PEM-encoded CA certificate file used to verify the Vault server's SSL certificate. This value, if exists, takes precedence over VAULT_CAPATH.`,
```
Comment (nixpanic) [inline frontend/packages/ceph-storage-plugin/src/components/modals/advanced-kms-modal/advanced-kms-modal.tsx:64]: Do we support setting `VAULT_CAPATH`? That would mean the PATH is inside the Ceph-CSI provisioner and node-plugin, and we normally do not add additional files to our containers. Maybe drop _This value, if exists, takes precedence over VAULT_CAPATH._?

**Review by nixpanic — APPROVED** [review]

**Review by nixpanic — APPROVED** [review]
Text and Secret look good to me. Thanks!

**Pull request merged**